src/derivation.test.ts

```diff
@@ -234,70 +233,62 @@ describe('derivation', () => {
       for (const input of inputs) {
         // eslint-disable-next-line no-loop-func
         await expect(
-          bip32Derive(
-            input as any,
-            Buffer.allocUnsafe(32).fill(1),
-            undefined,
-            Buffer.allocUnsafe(32).fill(1),
-          ),
+          bip32Derive({
+            path: input,
+            privateKey: Buffer.alloc(32).fill(1),
+            chainCode: Buffer.alloc(32).fill(1),
+          }),
         ).rejects.toThrow(
           'Invalid BIP-32 index: The index must be a non-negative decimal integer less than 2147483648.',
         );
       }
 
       await expect(
-        bip32Derive(`44'`, undefined, undefined, Buffer.alloc(32, 1)),
+        bip32Derive({
+          path: `44`,
+          chainCode: Buffer.alloc(32).fill(1),
+        }),
       ).rejects.toThrow(
         'Invalid parameters: Must specify either a parent private or public key.',
       );
-
-      await expect(
```

Comment:
Do we not want to test this anymore?

Reply:
It's tested in a separate `it` block below, so I removed the duplicate test here.